src/components/AssociatedResourceList.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Alert,
+  AlertGroup,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Title,
+} from '@patternfly/react-core';
+import {
+  K8sResourceKind,
+  ResourceLink,
+  useK8sWatchResources,
+  VirtualizedTable,
+  TableData,
+  RowProps,
+  TableColumn,
+  WatchK8sResource,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { SearchIcon } from '@patternfly/react-icons';
+import { getStatusLabel } from '../utils/statusLabel';
+
+type AssociatedResourceListProps = {
+  resource: K8sResourceKind;
+};
+
+const AssociatedResourceList: React.FC<AssociatedResourceListProps> = ({ resource }) => {
+  const { t } = useTranslation('plugin__kuadrant-console-plugin');
+
+  // TODO: dynamic?
```

Comment:
Going to leave this as-is for now. I found some (undocumented) APIs in the plugin SDK which look like they do API discovery, similar to what we're doing in topology. Will look into this in a separate issue.